channels/api.py

```diff
@@ -32,6 +32,23 @@
 User = get_user_model()
 
 
+def get_or_create_user(username):
+    """
+    Get or create a user on reddit using our refresh_token plugin
```

Comment:
nit: I would add some comments about the fact that this relies on our modified version of the reddit code

Reply:
done